src/todo/UI.lua

```diff
@@ -2,11 +2,14 @@
 function todo.create_minimized_button(player)
     todo.log("Creating Basic UI for player " .. player.name)
 
-    if not todo.get_maximize_button(player) and not todo.get_main_frame(player) and todo.show_minimized(player) then
+    if (not todo.get_maximize_button(player)
+      and not todo.get_main_frame(player)
+      and todo.show_button(player) ) then
```

Comment:
I thought this was easier to read, but maybe just me.

Reply:
Works for me :)